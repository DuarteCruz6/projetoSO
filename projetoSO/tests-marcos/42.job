WAIT 3000
WRITE [(i0,97)(e7,9)(a6,64)(g3,56)]
WRITE [(f6,2)(v0,97)(t7,48)(l7,29)]
WAIT 3000
SHOW
DELETE [e7]
WRITE [(y5,63)(k3,16)(k1,32)(q3,88)(n9,53)]
DELETE [f6,n9,i0]
WRITE [(n3,78)(x1,66)(c2,10)(d8,36)]
READ [g3]
WRITE [(j9,24)(y3,55)(z6,24)(n1,75)(y4,67)]
READ [x1]
READ [y3]
WRITE [(b2,13)(k5,72)(r2,31)(e9,55)]
DELETE [l7,y5,k5]
BACKUP
WRITE [(z9,28)(f1,2)(e3,17)(c1,62)(l3,13)]
WRITE [(y5,8)(l7,96)]
WRITE [(b6,39)(j4,98)(e8,55)(k8,41)]
BACKUP
WRITE [(h5,43)(n4,85)]
DELETE [q3,j9]
READ [c2]
WRITE [(u9,37)(r5,78)]
READ [v0]
DELETE [n4,v0,y4]
READ [j4]
READ [x1]
WRITE [(j9,73)(c6,38)(t3,34)(w2,94)]
READ [z9]
DELETE [e3,l3]
WRITE [(c3,91)]
READ [w2]
DELETE [l7]
READ [b2]
READ [y5]
BACKUP
SHOW
