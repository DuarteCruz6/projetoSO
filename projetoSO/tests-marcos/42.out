(a6, 64)
(e7, 9)
(f6, 2)
(g3, 56)
(i0, 97)
(l7, 29)
(t7, 48)
(v0, 97)
[(g3,56)]
[(x1,66)]
[(y3,55)]
[(c2,10)]
[(v0,97)]
[(j4,98)]
[(x1,66)]
[[(e3,KVSMISSING)(l3,KVSMISSING)(z9,28)]
[(w2,94)]
]
[(y5,8)]
(a6, 64)
(b6, 39)
(b2, 13)
(c3, 91)
(c6, 38)
(c1, 62)
(c2, 10)
(d8, 36)
(e8, 55)
(e3, 17)
(e9, 55)
(f1, 2)
(g3, 56)
(h5, 43)
(j9, 73)
(j4, 98)
(k8, 41)
(k1, 32)
(k3, 16)
(l7, 96)
(l3, 13)
(n1, 75)
(n3, 78)
(r5, 78)
(r2, 31)
(t3, 34)
(t7, 48)
(u9, 37)
(w2, 94)
(x1, 66)
(y5, 8)
(y3, 55)
(z9, 28)
(z6, 24)
