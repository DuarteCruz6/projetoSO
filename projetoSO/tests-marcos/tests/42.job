WRITE [(y9,31)(g4,82)]
READ [g4]
WRITE [(j7,49)(o9,1)(w5,87)]
WAIT 2000
WRITE [(q6,38)(q8,76)(h7,27)(q8,40)]
DELETE [h7,g4,y9]
DELETE [j7,q6,q8,o9,w5]
WRITE [(x1,77)]
WRITE [(i6,81)(s1,18)]
WRITE [(e2,96)(o8,62)]
READ [o8]
READ [i6]
DELETE [e2,o8,i6,x1,s1]
WAIT 3000
WRITE [(w0,29)(s9,30)(j1,27)(t4,80)]
WRITE [(z9,38)(q2,34)]
WAIT 2000
