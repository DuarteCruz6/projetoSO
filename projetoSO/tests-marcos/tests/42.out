[(g4,82)]
[(o8,62)]
[(i6,81)]
